Module.yaml

```diff
@@ -49,15 +49,14 @@ environments:
   - key: PARITY
     description: Enable parity checking (optional)
     value: ''
-    default: PARITY_NONE
-    type: enum
-    options:
-      - 'N'
-      - 'E'
-      - 'O'
-      - 'M'
-      - 'S'
+
+    default: 'None'
+    type: string
     name: Parity
+    options:
+    - 'None'
+    - 'Even'
+    - 'Odd'
```

Comment:
what about the other two values?

Reply:
we will not use the 9 bits so , it does not matter to use “Mark/Space” Christos informed is not required.